<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InscriptionDialog</class>
 <widget class="QDialog" name="InscriptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nom d'utilisateur</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_username"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_name"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="prNomLabel">
       <property name="text">
        <string>Pr√©nom</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_forname"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="adresseLabel">
       <property name="text">
        <string>Adresse</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="le_address"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="rIBLabel">
       <property name="text">
        <string>RIB</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="le_rib"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="l_errorInfos">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="motDePasseLabel">
       <property name="text">
        <string>Mot de passe</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_password">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="confirmerMotDePasseLabel">
       <property name="text">
        <string>Confirmer mot de passe</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_confirmePassword">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="l_errorPassword">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="b_inscription">
     <property name="styleSheet">
      <string notr="true">background-color: green;
color: white;</string>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
